dataset/src/main/java/org/keycloak/benchmark/dataset/DatasetResourceProvider.java

```diff
@@ -972,12 +979,49 @@ private void createClients(RealmContext context, Task task, KeycloakSession sess
         task.debug(logger, "Created %d clients in realm %s", context.getClientCount(), context.getRealm().getName());
     }
 
-    private void createGroups(RealmContext context, int startIndex, int endIndex, KeycloakSession session) {
+    private String getGroupName(boolean hiearchicalGroups, int hierarchyDepth, int countGroupsAtEachLevel, String prefix, int currentCount) {
```

Comment:
The parameter `hierarchyDepth` is not used, please review/test the code.

Reply:
Hi @ahus1
Thank you for the review and the fixes. Yes "." is tricky in regex. I removed the unused parameter and a typo.